libs/ui/src/components/icon-button-styled/IconButtonStyled.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import IconButton from '@mui/material/IconButton';
+import { ReactElement } from 'react';
+
+interface IconButtonStyled {
```

Comment:
extend `IconButtonProps` from `@mui/material/IconButton`
and spread props line 16:
```tsx
<_IconButtonStyled  {...props} />
```